### PR TITLE
AUT-3612: use real names of audit events in code base

### PR DESCRIPTION
**Pull request by LazarAlexandru-Constantin**
## What

Remove the code that does the concatenation and refactor the enum to have the full and correct names.

## Why

Having the "AUTH_" string concatenated at the beginning has some drawbacks:

- This piece of functionality is hidden away and makes it difficult to see how the data for the audit event is actually produced

- We need to remove the prefix from event names we might copy from logs, such as in Splunk, if we want search our code base.

- Some events have been created with the prefix added twice.

## How to review

Code Review

## Checklist

- [ ] Impact on orch and auth mutual dependencies has been checked.

- [X] No UCD review needed: no changes to the user interface

### CODE REVIEW

**Review by andrew-moores — APPROVED** [review]

**Review by ethanmills — DISMISSED** [review]
It looks like you've changed the orch code that adds the prefix, but not added the prefix to any of the Orch events?

File: orchestration-shared-test/src/main/java/uk/gov/di/orchestration/sharedtest/helper/AuditAssertionsHelper.java [hunk 1]
```diff
@@ -29,7 +29,7 @@ public static void assertNoTxmaAuditEventsReceived(SqsQueueExtension txmaAuditQu
     public static void assertTxmaAuditEventsReceived(
             SqsQueueExtension queue, Collection<AuditableEvent> events) {
 
-        var txmaEvents = events.stream().map(Objects::toString).map("AUTH_"::concat).toList();
+        var txmaEvents = events.stream().map(Objects::toString).toList();
```
[the remark this answers is not in the document]
Comment (andrew-moores) [reply orchestration-shared-test/src/main/java/uk/gov/di/orchestration/sharedtest/helper/AuditAssertionsHelper.java:32]: @ethanmills Is this a change you want in your code as well?  If not then we could create auth specific version of this and TxmaAuditEvent.java

[the remark this answers is not in the document]
Comment (ethanmills) [reply orchestration-shared-test/src/main/java/uk/gov/di/orchestration/sharedtest/helper/AuditAssertionsHelper.java:32]: There already is an auth specific version. This is the Orch version. For this change I'd suggest backing out any changes to `orchestration-shared` and `orchestration-shared-test` then I'd be happy to unblock it. We can pick up any changes to Orch separately

[the remark this answers is not in the document]
Comment (ethanmills) [reply orchestration-shared-test/src/main/java/uk/gov/di/orchestration/sharedtest/helper/AuditAssertionsHelper.java:32]: The Auth version is https://github.com/govuk-one-login/authentication-api/pull/5117/files/7ac7086043e54b3258652d6ded9b0278fa4a95a0#diff-408b8c9cfd0ddae3b2303df35759ad02868e01209df63ffd4f2f8609b37c5a74L32

I'd suggest adding an assertion that the names of the events start with `AUTH_`, but it's up to you

[the remark this answers is not in the document]
Comment (LazarAlexandru-Constantin) [reply orchestration-shared-test/src/main/java/uk/gov/di/orchestration/sharedtest/helper/AuditAssertionsHelper.java]: Rechecked all of the files changed and reverted a change to the AuditService Test which was sitting with orch.

Now, all of the changes should sit with auth.

**Review by andrew-moores — APPROVED** [review]

**Pull request merged**